source/analytic-geometry/sections/subsections/sec-hyperbolas/subsec-definition-of-a-hyperbola.ptx

```diff
@@ -3,162 +3,165 @@
     <definition xml:id="def-definition-of-a-hyperbola">
         <statement>
             <p>
-                Let F and <m>\text{F} \textbf{′}</m> be two fixed points in the plane. A <term>hyperbola</term> is the set of all points in the plane such that the difference of the distance of each point from F and F<term>′</term> is constant. We shall denote the constant by 2<m>a</m>, for some <m>a</m> <m>\</m> 0. That is, a point P is on the hyperbola if and only if |PF<term>′</term>| – |PF| = 2<m>a</m> (or |PF|– |PF<term>′</term>| = 2<m>a</m> , whichever is positive). The two fixed points F and F<term>′</term> are called the foci of the hyperbola.
+                Let F and <m>\text{F} \textbf{′}</m> be two fixed points in the plane. A <m>\textbf{hyperbola}</m> is the set of all points in the plane such that the difference of the distance of each point from <m>\text{F}</m> and <m>F′</m> is constant. We shall denote the constant by <m>2a</m>, for some <m>a  &gt;0</m> . That is, a point P is on the hyperbola if and only if <m>|PF\textbf{′}| – |PF| = 2a</m> (or <m> |PF|– |PF\textbf{′}| = 2a</m> , whichever is positive). The two fixed points <m>\text{F}\, \text{and}\, \text{F} \textbf{′}</m> are called the foci of the hyperbola.
```

Comment:
`<m>\textbf{hyperbola}</m>`: New terminology should use `<term>` (the way it was before was good), don't put normal text in LaTeX. There are a few instances of this, otherwise this looks good.